packages/esbuild/esbuild.bzl

```diff
@@ -367,12 +380,11 @@ def esbuild_macro(name, output_dir = False, splitting = False, **kwargs):
     deps = kwargs.pop("deps", []) + ["@esbuild_npm//esbuild"]
     entry_points = kwargs.get("entry_points", None)
 
-    args = kwargs.pop("args", {})
     args_file = kwargs.pop("args_file", None)
+    if args_file:
+        fail("Setting 'args_file' is not supported, set 'config_file' instead")
```

Comment:
todo/issue filed to remove the attribute later?

Reply:
Yeah, we could remove this in 5.0 - although it's kinda nice not to have all the verbosity of the config file just for one extra config option a user may want to pass.
What I really want here is a way to have the attribute private, but still settable via the macro, therefore hiding it from the docs.